README.md

```diff
@@ -31,9 +30,61 @@ But it is strongly recommended to only use [GitHub Actions "Release to Maven Cen
 
 ## License
 
-License is Apache-2. Each source file must include the Apache-2 header (build will fail otherwise).
+License is GNU General Lesser Public License (LGPL) version 3.0. Each source file includes the LGPL-3 header (build will fail otherwise).
 To update source files with the proper header, simply execute the below command:
 
 ```bash
 mvn license:update-file-header
 ```
+
+## Run IPMI Client inside Java
```

Comment:
Remove this section (it's already in the project documentation), and add a big link at the top of README.md to the site documentation and Javadoc.